build-utils/sentry-instrumentation.js

```diff
@@ -10,9 +10,8 @@ const {
   SENTRY_WEBPACK_WEBHOOK_SECRET,
   GITHUB_SHA,
   GITHUB_REF,
-  CI,
 } = process.env;
-const IS_CI = !!CI;
+const IS_CI = !!GITHUB_SHA;
```

Comment:
<3

Reply:
Shouldn't this be `GITHUB_REF` as the connection is through this env var below. Also I wonder if we really need `IS_CI`? Why not just check `GITHUB_REF` directly below?

Reply:
We use `IS_CI` elsewhere, https://github.com/getsentry/sentry/pull/22034/files#diff-bb490b7bd0a004bce840fd8ce6b94911de263f1eb455491f30596ea4a0aee48eR76

1) `GITHUB_REF` [is not guaranteed to exist](https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables) and 2) we currently only use `GITHUB_REF`, but we could add other Sentry tags later

Reply:
Eh, okay. Then that usage becomes GHA-only though (won't detect GCB as "ci env" correctly _if_ we wanted in the future).

I think this is fine right now, but I feel like there's a need to review and refactor all `CI` and `IS_CI` usages to make them accurate.

Reply:
Right, that's why it was using `CI` before